src/commands/bootcamp/bootcampNewTrack.ts

```diff
@@ -0,0 +1,80 @@
+import { Message, CategoryChannel, Permissions } from 'discord.js';
+import { CommandoClient, CommandoMessage } from 'discord.js-commando';
+import { AdminCommand } from '../../utils/commands';
+import { toTitleCase } from './utils';
+
+class BootcampNewTrackCommand extends AdminCommand {
+  constructor(client: CommandoClient) {
+    super(client, {
+      name: 'new-track',
+      aliases: ['add-track'],
+      group: 'bootcamp',
+      memberName: 'new-track',
+      args: [
+        {
+          key: 'category',
+          prompt: 'What will you name the new category?',
+          type: 'string'
+        }
+      ],
+      guildOnly: true,
+      description: 'Adds a new track category.'
+    });
+  }
+
+  async onRun(message: CommandoMessage, args: { category: string }): Promise<Message> {
+    let { category } = args;
+
+    category = toTitleCase(category);
+    const guild = message.guild;
+
+    const waitingRooms = <CategoryChannel>(
+      guild.channels.cache.find((channel) => channel.name.startsWith('Waiting Room') && channel.type === 'category')
+    );
+
+    if (!waitingRooms) {
+      guild.channels.create('Waiting Room', {
```

Comment:
should have a constant for `Waiting Room` since you are using this string literal more than once.